api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -506,10 +506,10 @@ type HostedClusterSpec struct {
 	// Max is 6 to account for OIDC;OVNSbDb for backward compatibility though they are no-op.
 	//
 	// +kubebuilder:validation:MaxItems=6
-	// +kubebuilder:validation:MinItems=4
+	// +kubebuilder:validation:MinItems=3
```

Comment:
the e2e is legitimately faling because we validate at least 4 items are set.
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_hypershift/5782/pull-ci-openshift-hypershift-main-e2e-aws/1897784933695361024

Maybe we can make this platform specific, something like

self.platform == "IBM" ? size(self.services) >= 3 : size(self.services) >= 4